src/main/java/com/faforever/client/chat/ChannelTabController.java

```diff
@@ -210,7 +210,11 @@ private void filterChatUserControlsBySearchString() {
         synchronized (chatUserControlMap) {
           for (Map.Entry<Pane, ChatUserItemController> chatUserControlEntry : chatUserControlMap.entrySet()) {
             ChatUserItemController chatUserItemController = chatUserControlEntry.getValue();
```

Comment:
not actually part of your PR, but while you're at it, the for loop and `getValue()` call can probably just be replaced by an interation over the valueSet.

(either repalce the for loop with `chatUserControlMap.values()` or maybe even just use the stream API and do):
```
chatUserControlMap.values().forEach(chatUserItemController -> {
    ...
});
```

Reply:
Done :) .